### PR TITLE
polkagate-rubic-hedera-alephzero-safeheron-defiarmor

### PR DESCRIPTION
**Pull request by mario-christopher**
This closes:
#480 
#481 
#482 
#483 
#484 
#457 

**Note:** Rubic Snap builder doesnt want the Snap to be displayed in Directory, so I did not add the `category` property for it.

### CODE REVIEW

File: src/registry.json [hunk 1]
```diff
@@ -1960,9 +1963,9 @@
         "name": "Aleph Zero Wallet",
         "author": {
           "name": "Bide",
-          "website": "https://bide.dev"
+          "website": "https://azero.dev/"
```
Comment (Montoya) [inline src/registry.json:1966]: This change does not make sense. Is the developer Bide? If so, this should be Bide's website.

File: src/registry.json [hunk 2]
```diff
@@ -2078,6 +2087,67 @@
...
+    "npm:@polkagate/snap": {
+      "id": "npm:@polkagate/snap",
+      "metadata": {
+        "name": "PolkaGate",
+        "author": {
+          "name": "PolkaGate",
+          "website": "https://polkagate.xyz/"
+        },
+        "website": "https://apps.polkagate.xyz/",
+        "summary": "Unlock the Power of Polkadot. Seamlessly manage accounts, connect to Polkadot dApps, and sign transactions.",
```
Comment (Montoya) [inline src/registry.json:2099]: Regardless of how the developer writes "dapp," we always spell it "dapp."

File: src/registry.json [hunk 3]
```diff
@@ -2078,6 +2087,67 @@
...
+        "description": "The Polkagate Snap is a MetaMask snap designed to enable MetaMask to interact with tokens within the Polkadot ecosystem, including Polkadot, Kusama, and more.\n\nTo use the snap, you must first install MetaMask and create your initial account on Ethereum. This will generate a recovery phrase, which the Polkagate snap will utilize to derive accounts for the Polkadot ecosystem.\n\nAfter MetaMask installation, manage your Polkadot ecosystem accounts and perform various transactions by installing the Polkagate snap. Users can install snaps by visiting the MetaMask Snaps Directory or each snap's specific dApp, which for the Polkagate snap the dapp address is https://apps.polkagate.xyz/ where users can manage transactions such as fund transfers, staking, participating in crowdloans, and more.",
```
Comment (Montoya) [inline src/registry.json:2100]: is a MetaMask **Snap**...

which the Polkagate **Snap** will utilize... 

by installing the Polkagate **Snap**.

> Users can install snaps by visiting the MetaMask Snaps Directory or each snap's specific dApp, which for the Polkagate snap the dapp address is https://apps.polkagate.xyz/ where users can manage transactions such as fund transfers, staking, participating in crowdloans, and more.

This sentence does not make sense in the context of our directory.

File: src/registry.json [hunk 4]
```diff
@@ -2078,6 +2087,67 @@
...
...re to pick the most cost-efficient route for your token swap by actively comparing swap rates across 200+ DEXs simultaneously – all within your MetaMask wallet.\n\nHere's how it works (YT tutorial link to be added later):\n\n1. Initiate your transaction on a DEX in a browser.\n\n2. Within the MetaMask transaction confirmation window, you'll have access to Rubic's Snap tab.\n\n3. Rubic's innovative 'Best Rate Finder' algorithm searches from over 200 DEXs, analyzing various routes to identify the most cost-efficient option for your intended swap.\n\n4. If Rubic indicates a better rate than the one offered by the current DEX you're on, you'll have the option to seamlessly switch to Rubic's App to complete the transaction using the more optimal route.\n\nStop juggling between multiple DEXs to find the best rate. Leave that heavy lifting to us and have peace of mind knowing you're always getting the best swap rate possible!\n\nAbout Rubic:\n\nRubic aggregates 70+ blockchains and testnets, while it enables swaps of 15,500+ assets with the best rates, highest liquidity, and transaction speeds — in one click, thanks to the integration of 220+ DEXs and bridges.\n\nhttps://rubic.exchange/ ",
```
Comment (Montoya) [inline src/registry.json:2131]: > (YT tutorial link to be added later)

This part should be removed

**Review by Montoya — CHANGES_REQUESTED** [review]

File: src/registry.json (hunk 1 above)
Comment (mario-christopher) [reply src/registry.json]: Done.

File: src/registry.json (hunk 2 above)
Comment (mario-christopher) [reply src/registry.json]: Done.

File: src/registry.json (hunk 3 above)
Comment (mario-christopher) [reply src/registry.json]: Done.

File: src/registry.json [hunk 5]
```diff
@@ -1960,9 +1963,9 @@
         "name": "Aleph Zero Wallet",
         "author": {
           "name": "Bide",
-          "website": "https://bide.dev"
+          "website": "https://bide.dev/"
         },
-        "website": "https://bide.dev",
+        "website": "https://azero.dev/",
```
Comment (FrederikBolding) [inline src/registry.json:1968]: Is it intentional to bring users to a block explorer after installing the Snap?

File: src/registry.json [hunk 6]
```diff
@@ -2212,7 +2282,7 @@
           },
           {
             "auditor": "Least Authority",
-            "report": "https://leastauthority.com/wp-content/uploads/2024/01/Safeheron_Crypto_Suites__Multiparty_ECDSA_Updated_Final_Audit_Report_Least_Authority.pdf"
+            "report": "https://leastauthority.com/wp-content/uploads/2024/02/Safeheron_Crypto_Suites__Multiparty_ECDSA_Updated_Final_Audit_Report_Least_Authority.pdf "
```
Comment (FrederikBolding) [inline src/registry.json:2285]:
```suggestion
            "report": "https://leastauthority.com/wp-content/uploads/2024/02/Safeheron_Crypto_Suites__Multiparty_ECDSA_Updated_Final_Audit_Report_Least_Authority.pdf"
```

**Comment by FrederikBolding** [comment]
@mario-christopher Can we split these into smaller chunks of PRs in the future? It feels unfair to block updates waiting for formatting changes on other Snap metadata

File: src/registry.json (hunk 5 above)
Comment (mario-christopher) [reply src/registry.json:1968]: It was an Update request from the Builder to have this as the `website` for the Snap.

File: src/registry.json (hunk 6 above)
Comment (mario-christopher) [reply src/registry.json]: Done.

File: src/registry.json (hunk 5 above)
Comment (Montoya) [reply src/registry.json:1968]: It is, in fact, intentional

**Review by FrederikBolding — APPROVED** [review]

**Pull request merged**